### PR TITLE
Fix Bug 1456477 - remove border (inset shadow) from top sites icons

### PR DESCRIPTION
**Pull request by rlr**

### CODE REVIEW

File: system-addon/content-src/components/TopSites/_TopSites.scss [hunk 1]
```diff
@@ -150,6 +150,10 @@ $half-base-gutter: $base-gutter / 2;
     border-radius: $top-sites-border-radius;
     box-shadow: inset $inner-box-shadow;
     position: absolute;
+
+    .dark-theme & {
+      box-shadow: none;
+    }
```
Comment (rlr) [inline system-addon/content-src/components/TopSites/_TopSites.scss]: Alternatively, I could do this below which is probably more consistent with the rest of our theming? Thoughts?

```
diff --git a/system-addon/content-src/components/TopSites/_TopSites.scss b/system-addon/content-src/components/TopSites/_TopSites.scss
index 81c3859d..aed108e2 100644
--- a/system-addon/content-src/components/TopSites/_TopSites.scss
+++ b/system-addon/content-src/components/TopSites/_TopSites.scss
@@ -148,12 +148,8 @@ $half-base-gutter: $base-gutter / 2;
     background-position: center center;
     background-repeat: no-repeat;
     border-radius: $top-sites-border-radius;
-    box-shadow: inset $inner-box-shadow;
+    box-shadow: var(--newtab-topsites-icon-shadow);
     position: absolute;
-
-    .dark-theme & {
-      box-shadow: none;
-    }
   }

   .rich-icon {
diff --git a/system-addon/content-src/styles/_theme.scss b/system-addon/content-src/styles/_theme.scss
index 735ed60e..2362e304 100644
--- a/system-addon/content-src/styles/_theme.scss
+++ b/system-addon/content-src/styles/_theme.scss
@@ -52,6 +52,7 @@ body {

   // Top Sites
   --newtab-topsites-background-color: $white;
+  --newtab-topsites-icon-shadow: 0 0 0 1px var(--newtab-inner-box-shadow-color);
   --newtab-topsites-label-color: inherit;

   // Cards
@@ -106,6 +107,7 @@ body {

   // Top Sites
   --newtab-topsites-background-color: $grey-70;
+  --newtab-topsites-icon-shadow: none;
   --newtab-topsites-label-color: $grey-10-80;

   // Cards
```

Comment (rlr) [reply system-addon/content-src/components/TopSites/_TopSites.scss]: I think I'm leaning to keeping `.dark-theme * {}` stuff contained in _theme.scss 🤔

**Comment by rlr** [comment]
Before:
screen shot 2018-04-24 at 10 34 43 am

After:
screen shot 2018-04-24 at 10 32 21 am

File: system-addon/content-src/components/TopSites/_TopSites.scss (hunk 1 above)
Comment (Mardak) [reply system-addon/content-src/components/TopSites/_TopSites.scss]: Nod. Let's do the css variable for the shadow

File: system-addon/content-src/styles/_theme.scss [hunk 2]
```diff
@@ -7,6 +7,17 @@
...
+// scss variables related to the theme.
+$border-primary: 1px solid var(--newtab-border-primary-color);
+$border-secondary: 1px solid var(--newtab-border-secondary-color);
+$shadow-primary: 0 0 0 5px var(--newtab-card-active-outline-color);
+$shadow-secondary: 0 1px 4px 0 $grey-90-20;
+$input-border: solid 1px var(--newtab-textbox-border);
+$input-border-active: solid 1px var(--newtab-textbox-focus-color);
+$input-error-border: solid 1px $red-60;
+$input-error-boxshadow: textbox-shadow($red-60);
+$inner-box-shadow: 0 0 0 1px var(--newtab-inner-box-shadow-color);
```
Comment (Mardak) [inline system-addon/content-src/styles/_theme.scss]: I suppose while we're touching these... ;) Sort the list alphabetically and make the borders consistently `1px solid <color>`

**Comment by as-pine-proxy** [comment]
[Treeherder: dc747923b63fa12e15fc6610e0614e26d8c788a9](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=dc747923b63fa12e15fc6610e0614e26d8c788a9)

**Pull request merged**